test/mocks/server/admin.cc

```diff
@@ -0,0 +1,30 @@
+#include "admin.h"
+
+#include <string>
+
+#include "envoy/admin/v3/server_info.pb.h"
+#include "envoy/config/core/v3/base.pb.h"
+
+#include "common/singleton/manager_impl.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using testing::_;
+using testing::Invoke;
+using testing::Return;
+using testing::ReturnPointee;
+using testing::ReturnRef;
+using testing::SaveArg;
```

Comment:
We're not using any of these using declarations in this file, so they can be deleted. This is part of why the clang-tidy check is failing.